Docker installation manual.md

```diff
@@ -0,0 +1,116 @@
+##Docker and Docker Compose installation manual
+
+Source website:
+	https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/
+
+**0. Open terminal.**
+
+**1. Uninstall old versions of Docker:**
+
+	$ sudo apt-get remove docker docker-engine docker.io
+
+**2. Update the apt package index:**
+
+	$ sudo apt-get update
+
+**3.Install packages to allow apt to use a repository over HTTPS:**
+
+	$ sudo  apt-get install \
+         	apt-transport-https \
+    		ca-certificates \
+    		curl \
+    		software-properties-common
+
+**4. Add Docker’s official GPG key:**
+
+	$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+	Alternative way(in case of emergency):
+
+Open following URL in browser(Chrome, if possible): https://download.docker.com/linux/ubuntu/gpg  
+
+	$ cat DownloadedGpgFilePath | sudo apt-key add - 
+
+**5. Verify that you now have the key using last 8 characters of the fingerprint:**
+
+	$ sudo apt-key fingerprint 0EBFCD88
+
+You should get sth like this:
```

Comment:
I wouldn't stick with this kind of abbreviations, might be unclear for some people. ;-)